RestClient.cpp

```diff
@@ -146,8 +130,12 @@ void RestClient::setContentType(const char* contentTypeValue){
     contentType = contentTypeValue;
 }
 
-void RestClient::setSSL(int _ssl){
-    ssl = (_ssl) ? 1 : 0;
+void RestClient::setSSL(bool _ssl){
```

Comment:
Changing this signature can break a lot of existing users.  Why not leave as an integer?